magda-gateway/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM node:10
-
 RUN mkdir -p /usr/src/app
 COPY . /usr/src/app
+RUN rm -Rf /usr/src/app/node_modules/bcrypt && cd /usr/src/app/component && npm install bcrypt
```

Comment:
This could use a comment explanation. I assume it needs linux binaries or something like that?

Reply:
Added comments above